com.unity.multiplayer.mlapi/Runtime/Core/NetworkUpdateLoop.cs

```diff
@@ -360,5 +384,92 @@ private static void Initialize()
 
             PlayerLoop.SetPlayerLoop(customPlayerLoop);
         }
+
+#if UNITY_EDITOR
+        private static void UninjectSystems()
```

Comment:
We might want to comment on what this is doing and why this is important?

Reply:
I did comment in the `Initialize()` method above, do you think that's not enough?